### PR TITLE
Create Luce Carević.json

### PR DESCRIPTION
**Pull request by luce-carevic**
👋

### CODE REVIEW

File: data/members/Luce Carević.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "Luce Carević",
+  "url": "https://luce.carevic.eu/fr",
+  "employment": {
+    "hiring": false,
+    "seeking": false
+  },
+  "linkedin": "https://www.linkedin.com/in/lucecarevic/",
+  "mastodon": "https://mastodon.design/@luce",
+  "rss": "https://luce.carevic.eu/fr/flux",
+  "twitter": "null"
```
Comment (ericwbailey) [inline data/members/Luce Carević.json:12]:
```suggestion
  "twitter": null
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Welcome! `null` can't be in quotes, otherwise it will get interpreted as a string. If you accept my suggestion it'll patch things up and you're good to be merged! 🙌

**Comment by luce-carevic** [comment]
Sorry for the mistake! 😅

**Review by ericwbailey — APPROVED** [review]
No worries! Welcome 🙌

**Pull request merged**